### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by robheymann** [element-plus/element-plus#2002] [Q&A] [answered]
Hi,

Is there any planned support for a dark mode theme?

**Comment by iamkun** [comment, marked as answer]
![v2-1e85fd955b16e3866418e5e83e82846d_r](https://user-images.githubusercontent.com/17680888/118219401-ff443700-b4ab-11eb-8a48-6c1e51c19240.png)


Dark Mode is on our roadmap.

**Reply by robheymann** [reply]
Awesome thanks for the great components!

**Reply by JenuelDev** [reply]
awesome, I will be waiting for the dark and light theme modes in element, it will be more popular.,, any update about this?

**Comment by ExtReMLapin** [comment]
https://github.com/ExtReMLapin/element-plus-darktheme

**Comment by sxzz** [comment]
#3754
